doc/whatsnew/fragments/9443.feature

```diff
@@ -0,0 +1,4 @@
+A new `github` reporter has been added. This reporter automatically annotates code on GitHub's user interface
+with the messages found during linting. Use it with `pylint --output-format=github`.
```

Comment:
```suggestion
A new `github` reporter has been added. This reporter returns the output of `pylint` in the format that
Github can use to automatically annotate code. Use it with `pylint --output-format=github`.
```

Users will still need to run the linter in the right workflow so it is not fully automatic.